src/igvfd/schemas/treatment.json

```diff
@@ -67,6 +70,74 @@
             ],
             "comment": "temperature must be specified if temperature_units is specified"
         },
+        "amount": {
+            "comment": "amount and amount_units are not required if the treatment is depleted.",
+            "if": {
+                "properties": {
+                    "depletion": {
+                        "const": true
+                    }
+                }
+            },
+            "then": {
+                "not": {
+                    "required": [
+                        "amount_units"
```

Comment:
is not now the dependency misplaced? amount_units in amount and amount in amount_units?